ignite/handlers/checkpoint.py

```diff
@@ -630,14 +626,18 @@ class DiskSaver(BaseSaveHandler):
         dirname: Directory path where the checkpoint will be saved
         atomic: if True, checkpoint is serialized to a temporary file, and then
             moved to final destination, so that files are guaranteed to not be damaged
-            (for example if exception occurs during saving).
+            (for example if exception occurs during saving). Setting ``atomic=True`` is
+            recommended if ``n_saved=1`` is set in checkpoint object. For details, see
+            issue `#1986`_
```

Comment:
I think we can remove the link to the issue here. We can create a paragraph below describing essential points from the issue and add "see notes below" or something like that here.

Reply:
done